Index.html

```diff
@@ -9,28 +9,23 @@
 </head>
 <body>
    <div class="container-home">
-       <div class="text-block">
            <h4>made with love</h4>
            <p>for you</p>
            <div>
                <button class="button"><a class="arrow">&#8249;</a></button>
                <button class="button"><a class="arrow">&#8250;</a></button>
-           </div>
-       </div>
+           </div> 
    </div>
-   &nbsp;
-   <div>
+   <div class="container-image"   >
        <img class="backgroud-image" src="/images/about_img_2.png">
        <img class="top-image" src="/images/about_img_1.png">
    </div>
-   &nbsp;
-   <div>
+   <div class="container-about">
        <h1>about</h1>
        <h5>WE CREATE DELICIOUS MEMORIES</h5>
        <p>Renome it is renowned restaurant delights the senses with elegant ambiance, gracious service and delectable menus in the heart of Manhattan’s. The distinctive setting boasts contemporary furnishings and finishes atop the restaurant’s grand modern architecture. Renome offers the room for private dining where guests enjoy creative foods and sumptuous late night desserts. Is one of the few Manhattan restaurants to have a coveted 3-star Michelin rating.</p>
    </div>
-   &nbsp;
-   <div class="gallery-back">
+   <div class="container-gallery">
        <h1 class="gallery">gallery</h1>
```

Comment:
Probably should rename it more specific, not just gallery, else not seeing it in code would think that this class name is for all of gallery, not just text